src/components/Form.js

```diff
@@ -23,31 +24,31 @@ function Form() {
     setInput(input);
   };
   return (
-    <form>
-      <label htmlFor="label">
-        ADD NEW BOOK
-        <div>
-          <input
-            type="text"
-            onChange={getInputs}
-            name="title"
-            value={input.title}
-            placeholder="Book Title"
-          />
-          <br />
-          <input
-            type="text"
-            name="author"
-            onChange={getInputs}
-            value={input.author}
-            placeholder="Author"
-          />
-          <br />
-          <button type="submit" onClick={handleSubmit}>
-            ADD BOOK
-          </button>
-        </div>
-      </label>
+    <form className="book-form">
+      <h2 className="add-n-book">ADD NEW BOOK</h2>
+      <div className="input-wrapper">
+        <input
+          className="input-title"
+          type="text"
+          onChange={getInputs}
+          name="title"
+          value={input.title}
+          placeholder="Book Title"
+        />
+        <br />
+        <input
+          className="input-author"
+          type="text"
+          name="author"
+          onChange={getInputs}
+          value={input.author}
+          placeholder="Author"
+        />
+        <br />
+        <button className="add-button" type="submit" onClick={handleSubmit}>
+          ADD BOOK
+        </button>
```

Comment:
- [ ] Kindly use the correct font family for the `Add Book` button as per requirement.
![image](https://github.com/AbuTalha3/bookstore-app/assets/81932478/17fc2e91-37d0-49a4-a669-c28b5f565080)